addrreg/migrations/0006_auto_20170920_0842.py

```diff
@@ -0,0 +1,301 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please give this migration a more sensible name.

Reply:
Renamed.